layers/+lang/typescript/packages.el

```diff
@@ -89,7 +91,11 @@
                           (progn
                             (with-current-buffer outputbuf (erase-buffer))
                             (write-region nil nil tmpfile)
-                            (if (zerop (apply 'call-process "tsfmt" nil outputbuf nil (list tmpfile)))
+                            (if (zerop (apply 'call-process "tsfmt" nil outputbuf nil
+                                              (list (format
+                                                     "--baseDir='%s' --"
+                                                     default-directory)
```

Comment:
@aindlq  [tsfmt has interesting settings rules](https://github.com/vvakame/typescript-formatter#read-settings-rules)
In my tests it seems to pickup the closest `tsfmt.json` file on the specified path.  
Since  `default-directory ` is the directory of the current file, it allows user to have different  `tsfmt.json` files for different folders (overrides) if needed.